### ISSUE TITLE
QF - Show round information (including projects) after round is completed

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#3250]
It would be nice to show to future sponsors, users of QF etc. the results from past rounds. So it would be nice to have pages called something like "giveth.io/qf-closed-alpharound" or something like that after the round closed... that still shows all the projects that were in that round, and the banner w/ the sponsors etc.

But the gitcoin passport banner should say "This round has ended" on those pages.

wdyt? is this a crazy idea? @MohammadPCh 
<img width="1470" alt="Screenshot 2023-09-27 at 6 07 36 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/75490971/14094a92-122d-46bb-896a-06faac8fe8ba">

**Comment by GriffGreen** [comment]
YAY!! YES PLEASE!

**Comment by MohammadPCh** [comment]
hmm... My suggestion is to have a specific page for each round, and it's good to have a different UI for it. because when a round is finished, our focus is on the statistics not projects and actions.
wdyt @laurenluz @GriffGreen

**Comment by laurenluz** [comment]
yes!! actaully that would be amazing. we could perhaps put a little banner/thing (like we have on the homepage) that shows "number of unique donors" "amount of donations" and cool stats like that. 

and then we can show the project that were in the round, and it can have its own URL too.

does that sound good @MohammadPCh ? I can then work with Mo on this?

**Comment by laurenluz** [comment]
what if we just removed the title?

I think you nailed the most important stats! and I love this!

**Comment by laurenluz** [comment]
Also @mosaeedi - let's keep the original banner, we should show the sponsors

for the address we can use somethign like giveth.io/qf/archived/alpharound

**Comment by laurenluz** [comment]
@CarlosQ96 - I think we will need some backend support for this. we might need slug property added to the round, and endpoint to fetch data from the backend

**Comment by laurenluz** [comment]
Carlos things most of the info is available but there is no endpoint. need to make it public in the API... everything is saved int he QF round history.

the unique donors
the total amount

**Comment by mosaeedi** [comment]
[Link to Figma](https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=4543-40256&mode=design&t=jJ3VCVxD7e9SzLqb-4)

<img width="879" alt="Screenshot 2023-10-27 at 3 14 03 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/c00d48a7-9c57-44e3-9845-04266534ee07">

**Comment by laurenluz** [comment]
@mohammadranjbarz working on this in the backend to unblock Cherik
- creating a webservice for stats
- change existing web services to support qf round slug instead of qfround id

**Comment by laurenluz** [comment]
@mosaeedi I made some copy changes/small comments on the design in figma, can you have a look & resolve them?

**Comment by mosaeedi** [comment]
@laurenluz Everything looks good.

**Comment by laurenluz** [comment]
yess I put in the scaling QF spec that it would also be great if someone could upload the banners/components in admin bro for each round...

I agree we should store them in the backend somewhere.

looping in @rodricastillo9013 so he can weigh in on the standard. I think it makes sense how you suggested Cherik!

**Comment by rodricastillo9013** [comment]
We can define a standard. We can take the figma design as a reference here: https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=917-55316&mode=design&t=WzyseJyQF8IEqBcH-0

Width: 1440px
Height: 447 XP

We could define certain components of this banner and always work with the same quantity.

1- Background Image
2- Sponsors Right side
3- Extra Element 1 (Left)
4- Extra Element 2 (Top)
5- Extra Element (Bottom)

Are you ok with it? @mohammadranjbarz @laurenluz

**Comment by laurenluz** [comment]
@rodricastillo9013 - can we also make a standard for where 3-5 go on the design?

**Comment by rodricastillo9013** [comment]
Exactly, leave the positions predefined. @mohammadranjbarz How do you need position references? based on a design? with the locations of each element? as a reference wireframe?

**Comment by laurenluz** [comment]
where did we leave off with this @aminlatifi @MohammadPCh ?

**Comment by MohammadPCh** [comment]
@laurenluz  I'm currently waiting for the backend, and it seems like the frontend code is deprecated.

**Comment by MohammadPCh** [comment]
@laurenluz, Today I worked on this issue, and here are my updates:
This page is implemented (preview link):
[Example round](https://giveth-dapps-v2-git-add-archived-qf-page-givethio.vercel.app/qf-archive/59)
I have a question:
In older we show Project card in 2 situations:
1. It has been included in the active QF round so we show the estimated matching amount
2. It hasn't been included in any active QF round.
<img width="916" alt="Screenshot 2024-02-17 at 6 30 36 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/17097576/67fb5d35-ba8f-4b45-8b8f-95f5b52c0d3b">
are we gonna show the matched amount on this page on each project card? or leave it?
if we want to show it we need @mohammadranjbarz to provide it for FE.
besides that, these FE needs these values from the backend:

1. Round Middle banner title
2. Round Middle banner description
3. Round background address
4. Round sponsor's image address
5. Round top-left image address
6. Round top-right image address
7. Round bottom-left image address
8. Round bottom-right image address
9. round background color
10. round text color

that's it.
cc: @jainkrati

**Comment by laurenluz** [comment]
if it's not too much work... for the purposes of displaying projects in the archived round pages... I hope we can show the simple project card that we show usually when they are not in active round... whether they are also in an active round or not.

If that's too complex though, I think it's fine if we have to show the "project in an active round" card for some on the archived page..

these pages won't have high regular traffic anyway... they will just be there for reference & for sharing with possible future sponsors etc.

**Comment by MohammadPCh** [comment]
@laurenluz, yes, it's straightforward. I released a version in shadow yesterday along with the QF release. You can check it here: 
`https://giveth.io/qf-archive/[slug]`
for example:
https://giveth.io/qf-archive/buidl-on-polygon
As you can see, the banner is not okay. Additionally, we can hide the estimated matching section from the cards. Let me know what you think about it.

**Comment by laurenluz** [comment]
Hey @MohammadPCh! Yeah I think we need the banners stored in the backend, there is another issue for that (I think you are aware).

I think it would be better to hide the estimated matching on the project card when it shows in archived round pages... but we need to make sure that if that same project is in an active QF round, the estimated matching is not hidden from the views of that project card from the regular projects page, or the QF round page.

Also.. I see that we are showing the donation amount & unique donors here for polygon round... but we don't actually have that info yet for the poylgon round. maybe we need some field in admin bro where a round is marketed as completed/distributed... and then the unique donors shows up there...
because unique donors != unique donor addresses... some of those are sybils.

**Comment by jainkrati** [comment]
@mohammadranjbarz pls take this up post Superfluid

**Comment by RamRamez** [comment]
For this issue to be done, we need to store the banner in DB and banner is not just a simple image, sometimes it has several elements inside the main banner, so first we should define how many elements are going to be used in the banner and where they are going to be placed in the banner.
It's better to sync with the design team and make sure we are on the same page before adding new fields to the DB.
@MoeNick @jainkrati @mosaeedi Let's schedule a short call.

**Comment by RamRamez** [comment]
Please consider that this issue is related to #3665 
Mentioning @rodricastillo9013 to keep him in the loop

**Comment by mosaeedi** [comment]
We should only have 3 elements for the QF banner


1. Title 
2. Sponsors _(just a png image holder)_
3. background image _(Just an image without any text)_


Please take a look here to see where you should put them in different sizes.

[QF banner layout](https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=5897-26502&mode=design&t=Pwa1mhDdKdLeHqYH-4)

CC: @RamRamez @laurenluz @jainkrati

**Comment by laurenluz** [comment]
> We should only have 3 elements for the QF banner
> 
> 
> 1. Title 
> 2. Sponsors _(just a png image holder)_
> 3. background image _(Just an image without any text)_
> 
> 
> Please take a look here to see where you should put them in different sizes.
> 
> [QF banner layout](https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=5897-26502&mode=design&t=Pwa1mhDdKdLeHqYH-4)
> 
> 
> <img width="1027" alt="Screenshot 2024-04-15 at 12 21 43 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/64e1258b-3176-451b-b780-8d27c30bf309">
> 
> 
> 
> CC: @RamRamez @laurenluz @jainkrati 
> 

@mosaeedi we also have overlay images on the banner which are custom for each round and used for the responsive versions.

**Comment by laurenluz** [comment]
> We should only have 3 elements for the QF banner
> 
> 
> 1. Title 
> 2. Sponsors _(just a png image holder)_
> 3. background image _(Just an image without any text)_
> 
> 
> Please take a look here to see where you should put them in different sizes.
> 
> [QF banner layout](https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=5897-26502&mode=design&t=Pwa1mhDdKdLeHqYH-4)
> 
> 
> <img width="1027" alt="Screenshot 2024-04-15 at 12 21 43 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/64e1258b-3176-451b-b780-8d27c30bf309">
> 
> 
> 
> CC: @RamRamez @laurenluz @jainkrati 
> 

@mosaeedi we also have overlay images on the banner which are custom for each round and used for the responsive versions.

**Comment by RamRamez** [comment]
@laurenluz adding overly images make it very complicated to store in DB and show it properly in FE, because it's customized for each round and we need to store the position (top, left, right, bottom), size (width and height) and responsiveness config for each overlay image. Otherwise, we should standardize overlay images for all rounds.

**Comment by RamRamez** [comment]
I can imagine a solution that would be having different background images for each screen size and adding overlay images to the background image, so we have only a background image and no other elements.

**Comment by laurenluz** [comment]
Ahh gotcha. Ok let's just keep it simple with a background image. If we are able to upload this into admin bro for archived rounds... it will be easy enough to manage.

**Comment by Meriem-BM** [comment]
Hey @RamRamez, could you clarify what should I be working on here.

**Comment by RamRamez** [comment]
@maryjaf to test and verify

**Comment by Meriem-BM** [comment]
@maryjaf you can test this

**Comment by maryjaf** [comment]
> The passport banner isn't shown in this page, please take a look @Meriem-BM
> https://staging.giveth.io/qf-archive/16

Thanks @Meriem-BM 
It has been resolved and now this banner is shown
![image](https://github.com/Giveth/giveth-dapps-v2/assets/111529185/ef548a79-71a0-4a8b-9449-fbb9ccbb3045)

**Comment by maryjaf** [comment]
- [ ] 3-
There is a new problem, please take a look @Meriem-BM 
- There were 3 projects in roundId:16 : https://impact-graph.serve.giveth.io/admin/resources/QfRound/records/16/show
- in this page https://staging.giveth.io/qf-archive/16 
12 projects are shown, in fact all of projects in active current round(roundId:90) are shown in archive page of qf round Id:16 
![image](https://github.com/Giveth/giveth-dapps-v2/assets/111529185/3f21629e-bfe3-4128-ac46-1e12d44daa87)
![image](https://github.com/Giveth/giveth-dapps-v2/assets/111529185/4f455afa-cc29-4661-94cb-1ac900e47d2c)

**Comment by Meriem-BM** [comment]
@maryjaf you can test this again

**Comment by maryjaf** [comment]
> * There is a new problem, please take a look @Meriem-BM
> * There were 3 projects in roundId:16 : https://impact-graph.serve.giveth.io/admin/resources/QfRound/records/16/show
> * in this page https://staging.giveth.io/qf-archive/16
>   12 projects are shown, in fact all of projects in active current round(roundId:90) are shown in archive page of qf round Id:16

Thanks @Meriem-BM This problem also has been resolved,
___________
when this changes applied on production , for showing matching pool correctly  these new field in qf page also should be set for previous rounds. 
cc: @laurenluz 

![image](https://github.com/Giveth/giveth-dapps-v2/assets/111529185/6da9d78e-5f50-4f24-b861-910bc86c9c52)

**Comment by laurenluz** [comment]
cc @koday1

**Comment by koday1** [comment]
@laurenluz @maryjaf @Meriem-BM should this issue be closed now that the archived QF rounds tab is up and functioning well at Giveth.io/qf?

**Comment by laurenluz** [comment]
yep!
